WardenApi/src/main/java/com/salesforce/dva/warden/entity/Entity.java

```diff
@@ -0,0 +1,5 @@
+package com.salesforce.dva.warden.entity;
```

Comment:
Entities should not be in the API.  Delete com.salesforce.dva.warden.entity package and all its contents.